packages/vite/src/node/plugins/css.ts

```diff
@@ -675,7 +690,12 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
       await urlEmitQueue.run(async () =>
         Promise.all(
           urlEmitTasks.map(async (info) => {
-            info.content = await finalizeCss(info.content, true, config)
+            info.content = await finalizeCss(
+              info.content,
+              true,
+              config,
+              new Map([[info.originalFilename, info.end]]),
```

Comment:
`end` is a different value from what we want.

Reply:
https://github.com/vitejs/vite/blob/899d9b1d272b7057aafc6fa01570d40f288a473b/packages/vite/src/node/plugins/css.ts#L662-L668

Seeing from the code above, I thought it would be the right value. Do you think the `end` should be `getLineCount(info.index)`?

Reply:
Yes, it should be `getLineCount(info.content)`. `info.end` is position where `__VITE_CSS_URL__${}__` ends in JS.